test/unit/extent.js

```diff
@@ -278,6 +278,31 @@ describe('Extent', function () {
         assert.equal(toValues[3], withValues.south);
     });
 
+    it('should convert TMS extent values to string', function () {
+        const withValues = new Extent('TMS:4326', 0, 1, 2);
+        const tostring = withValues.toString(',');
+        const toValues = tostring.split(',').map(s => Number(s));
+        assert.equal(toValues[0], withValues.zoom);
+        assert.equal(toValues[1], withValues.row);
+        assert.equal(toValues[2], withValues.col);
+    });
```

Comment:
Why not compare it to `0,1,2` ?

Reply:
To test `toString`  with inverse function.